pdc_to_crdch_transformation.ipynb

```diff
@@ -0,0 +1,879 @@
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from linkml_runtime.dumpers import yaml_dumper\n",
+    "\n",
+    "print(yaml_dumper.dumps(example_observation_set))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Transform PDC Diagnosis into CRDCH Diagnosis\n",
+    "\n",
+    "We first define transformation methods for creating a CCDH BodySite from the `primary_site` in the PDC case, and also for transforming sample information from the case into a CCDH Specimen. Using these two methods we can define a combined diagnosis transformation method which can transform a full PDC diagnosis into a CCDH Diagnosis."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 86,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "id: df520aac-8f98-11ea-b1fd-0aad30af8a83\n",
+      "identifier:\n",
+      "- value: C3N-03889-DX\n",
+      "  system: PDC-submitter-id\n",
+      "condition:\n",
+      "  coding:\n",
+      "  - code: Squamous cell carcinoma, NOS\n",
+      "    system: https://example.org/PrimaryDiagnosis\n",
+      "metastatic_site:\n",
+      "- site:\n",
+      "    coding:\n",
+      "    - code: Head and Neck\n",
+      "      system: https://example.org/BodySite\n",
+      "stage:\n",
+      "- observations:\n",
+      "  - observation_type:\n",
+      "      coding:\n",
+      "      - code: Overall\n",
+      "        system: https://example.org/CancerStageObservation\n",
+      "    value_codeable_concept:\n",
+      "      coding:\n",
+      "      - code: Stage IV\n",
+      "        system: https://example.org/CancerStageObservation\n",
+      "  - observation_type:\n",
+      "      coding:\n",
+      "      - code: Clinical Metastasis (M)\n",
+      "        system: https://example.org/CancerStageObservation\n",
+      "    value_codeable_concept:\n",
+      "      coding:\n",
+      "      - code: M0\n",
+      "        system: https://example.org/CancerStageObservation\n",
+      "morphology:\n",
+      "  coding:\n",
+      "  - code: 8070/3\n",
+      "    system: https://example.org/Morphology\n",
+      "related_specimen:\n",
+      "- id: df502f10-8f98-11ea-b1fd-0aad30af8a83\n",
+      "  source_material_type: Primary Tumor\n",
+      "  general_tissue_morphology: Tumor\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "def create_body_site(site_name):\n",
+    "    \"\"\"Create a CCDH BodySite based on the name of a site in the human body.\"\"\"\n",
+    "    \n",
+    "    # Accept 'None'.\n",
+    "    if site_name is None:\n",
+    "        return None\n",
+    "    \n",
+    "    # Some body sites are not currently included in the CCDH model. We will need to translate these sites\n",
+    "    # into values that *are* included in the CCDH model.\n",
+    "    site_mappings = {\n",
+    "        'Larynx, NOS': crcdh_model.EnumCRDCHBodySiteSite.Larynx\n",
```

Comment:
shouldn't that be crdch_model, not crcdh_model?

Reply:
I did a global find and replace and the notebook ran to completion

Reply:
Great catch @turbomam. Thank you. I've changed the typo to the right spelling.